nomenklatura/store/sql.py

```diff
@@ -0,0 +1,171 @@
+from datetime import datetime
+from typing import Any, Generator, List, Optional, Set, Tuple
+
+from banal import as_bool
+from followthemoney.property import Property
+from sqlalchemy import Table, create_engine, delete, select
+from sqlalchemy.sql.selectable import Select
+
+from nomenklatura.dataset import DS
+from nomenklatura.db import (
+    DB_URL,
+    POOL_SIZE,
+    ensure_tx,
+    get_metadata,
+    get_statement_table,
+    get_upsert_func,
+)
+from nomenklatura.entity import CE
+from nomenklatura.resolver import Resolver
+from nomenklatura.statement import Statement
+from nomenklatura.store import Store, View, Writer
+
+
+def pack_statement(stmt: Statement) -> dict[str, Any]:
+    data: dict[str, Any] = stmt.to_row()
+    data["target"] = as_bool(data["target"])
+    data["external"] = as_bool(data["external"])
+    data["first_seen"] = data["first_seen"] or datetime.utcnow()
```

Comment:
I don't think we should make up dates somewhere in the middle of the pipeline. That would lead to pretty random outcomes. Maybe we make those columns nullable instead? 

Been struggling with the same issue: https://github.com/opensanctions/opensanctions/blob/main/zavod/zavod/tools/load_db.py#L42-L45